straxen/plugins/pulse_processing.py

```diff
@@ -17,6 +17,13 @@
              'See straxen.hit_min_amplitude for options.'
     )])
 
+HITFINDER_OPTIONS_he = tuple([
+    strax.Option(
+        'hit_min_amplitude_he',track=True,
+        default=tuple(np.ones(800, dtype=np.int16) * 15),
```

Comment:
I think you'll have to add this to the https://github.com/XENONnT/straxen/blob/petergaemers-patch-2/straxen/hitfinder_thresholds.py as:

1. Strax cannot save the metadata like this as the saver is complaining that we cannot json this
2. We also don't want 800 times 15 in the metadata.

Reply:
fixed